ethergo/submitter/submitter.go

```
@@ -676,18 +732,31 @@
 
 // getGasEstimate gets the gas estimate for the given transaction.
 // TODO: handle l2s w/ custom gas pricing through contracts.
-func (t *txSubmitterImpl) getGasEstimate(ctx context.Context, chainClient client.EVM, chainID int, tx *types.Transaction) (gasEstimate uint64, err error) {
+func (t *txSubmitterImpl) getGasEstimate(ctx context.Context, chainClient client.EVM, chainID int, tx *types.Transaction) (gasLimit_new uint64, err error) {
```

golangci-lint (1.62.2)

[warning] var-naming: don't use underscores in Go names; method result gasLimit_new should be gasLimitNew

(revive)